roles/common/tasks/main.yml

```diff
@@ -67,19 +67,29 @@
   command: hostnamectl set-hostname --static --transient {{ inventory_hostname }}
   tags: dns
 
-# setup base ssh and reject rules
-- name: setup ssh iptables rule
-  iptables:
-    port: 22
-    protocol: tcp 
-    comment: "ssh incoming"
+# setup base ssh rules
+- name: turn off firewalld
+  service:
+    name: firewalld
+    state: stopped
   tags: iptables
 
-# have to use command as there is no opt field in module yet.
-#- name: setup reject all
-#  command: iptables -A INPUT -j REJECT --reject-with icmp-host-prohibited
-#  tags: iptables
+- name: iptables start
```

Comment:
I tried it that way, iptables kept failing to load on the first time around.  It does seem redundent, but it I would rather it work each time then be a maybe